src/Sentry/Platforms/iOS/SentrySdk.cs

```diff
@@ -0,0 +1,135 @@
+            //         o.EnableAppLifecycleBreadcrumbs = options.Android.EnableAppLifecycleBreadcrumbs;
+            //         o.EnableSystemEventBreadcrumbs = options.Android.EnableSystemEventBreadcrumbs;
+            //         o.EnableUserInteractionBreadcrumbs = options.Android.EnableUserInteractionBreadcrumbs;
+            //         o.EnableUserInteractionTracing = options.Android.EnableUserInteractionTracing;
+            //         o.ProfilingTracesIntervalMillis = (int)options.Android.ProfilingTracesInterval.TotalMilliseconds;
+            //
+            //         // These options are in Java.SentryOptions but not ours
+            //         o.AttachThreads = options.Android.AttachThreads;
+            //         o.ConnectionTimeoutMillis = (int)options.Android.ConnectionTimeout.TotalMilliseconds;
+            //         o.Dist = options.Android.Distribution;
+            //         o.EnableNdk = options.Android.EnableNdk;
+            //         o.EnableShutdownHook = options.Android.EnableShutdownHook;
+            //         o.EnableUncaughtExceptionHandler = options.Android.EnableUncaughtExceptionHandler;
+            //         o.ProfilingEnabled = options.Android.ProfilingEnabled;
+            //         o.PrintUncaughtStackTrace = options.Android.PrintUncaughtStackTrace;
+            //         o.ReadTimeoutMillis = (int)options.Android.ReadTimeout.TotalMilliseconds;
+            //
+            //         // In-App Excludes and Includes to be passed to the Android SDK
+            //         options.Android.InAppExclude?.ToList().ForEach(x => o.AddInAppExclude(x));
+            //         options.Android.InAppInclude?.ToList().ForEach(x => o.AddInAppInclude(x));
+            //
+            //         // These options are intentionally set and not exposed for modification
+            //         o.EnableExternalConfiguration = false;
+            //         o.EnableDeduplication = false;
+            //         o.AttachServerName = false;
+            //
+            //         // These options are intentionally not expose or modified
+            //         //o.MaxRequestBodySize   // N/A for Android apps
+            //         //o.MaxSpans             // See https://github.com/getsentry/sentry-dotnet/discussions/1698
+            //
+            //         // Don't capture managed exceptions in the native SDK, since we already capture them in the managed SDK
+            //         o.AddIgnoredExceptionForType(JavaClass.ForName("android.runtime.JavaProxyThrowable"));
+
+        });
+
+        // Set options for the managed SDK that depend on the Cocoa SDK
+        // options.AddEventProcessor(new CocoaEventProcessor(cocoaOptions!));
+        options.CrashedLastRun = () => SentryCocoa.SentrySDK.CrashedLastRun;
+        // options.EnableScopeSync = true;
+        // options.ScopeObserver = new CocoaScopeObserver(options);
```

Comment:
Save to get rid of, there's no scope sync within the cocoa SDK
```suggestion
```

Reply:
Ok, I'll keep that in mind when I do the next set of these.  Leaving the comment there for now to remind me.  Thanks.